chat.simplex.simplex.yml

```diff
@@ -0,0 +1,54 @@
+app-id: chat.simplex.simplex
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: simplex
+separate-locales: false
+build-options:
+  strip: false
+  no-debuginfo: true
+  no-debuginfo-compression: true
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-download/simplex:create
+  - --talk-name=org.freedesktop.Notifications
```

Comment:
```suggestion
```

libnotify uses the poral inside sandboxed apps from 0.8.0

Reply:
Done.